nextcord/ext/help_commands/embedded.py

```diff
@@ -0,0 +1,97 @@
+from typing import TYPE_CHECKING
```

Comment:
```suggestion
from __future__ import annotations
from typing import TYPE_CHECKING
```

Reply:
Could you elaborate on the purpose of this suggested change?

Reply:
It's because you're using `TYPE_CHECKING`
BTW, that's also not needed

Reply:
@H4ckerxx44 if the annotations beta isnt enabled then python will store the typehint as **an object**.

`TYPE_CHECKING` is normally `False`, so python won't know whats `Mapping`? whats `Union`? python is not a mind-reading machine. 

annotations beta allows you to store it as string, so python will not emotionally damage you.

Reply:
The way it is is fine. While `TYPE_CHECKING` isn't *needed*, it is not used incorrectly since Mapping and Union are only used in type annotations.

Putting types in quotes is only necessary for partially initialized classes, such as class that's defined later in the file or the current class that the function is contained in.